HomeWork1PolishchukDariia.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cmath>
+#include <iostream>
+using namespace std;
+
+int main() {
+  setlocale(LC_ALL, "Russian");
+
+  double a{};
+  double b{};
+  double c{};
+  const int min{0};
+  const int max{1000};
```

Comment:
```suggestion
  constexpr int min{0};
  constexpr int max{1000};
```
вы же знаете значения этих констант на этапе компиляции поэтому вместо const лучше использовать constexpr